tests/release/e2e-test-default-with-deployment.go

```diff
@@ -0,0 +1,231 @@
+package release
+
+import (
+	"os"
+	"strings"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
+	"knative.dev/pkg/apis"
+
+	ecp "github.com/hacbs-contract/enterprise-contract-controller/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	klog "k8s.io/klog/v2"
+)
+
+var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deployment", Label("release", "withDeployment", "HACBS"), func() {
+	defer GinkgoRecover()
+	// Initialize the tests controllers
+	framework, err := framework.NewFramework()
+	Expect(err).NotTo(HaveOccurred())
+
+	var defaultEcPolicy = ecp.EnterpriseContractPolicySpec{}
+	var devNamespace = uuid.New().String()
+	var managedNamespace = uuid.New().String()
```

Comment:
please give the namespace like we do it in other [tests](https://github.com/redhat-appstudio/e2e-tests/blob/c1f0f6ffeae1edc8cc07b6afd8a42249160f861b/tests/e2e-demos/e2e-demo.go#L66), so it's easier to identify which namespace to investigate in case of failures in CI (or during test development)

Reply:
we wanted it to be unique on each run so it be different on each run thus won't be confusion if there is more one run in parallel , 
@davidmogar @mmalina  what do you say ?

Reply:
More runs in parallel? But each test run is always in a new cluster anyway, no? So it shouldn't be a problem. And if other tests define a name, I think we should do the same.

Reply:
@kasemAlem take a look at the method I linked in the comment. It will generate a namespace name with the prefix you specify + 4 random chars. This is usually sufficient even if you run the test multiple times against a single cluster.